src/Interfaces/RequestSender.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AshAllenDesign\LaravelExchangeRates\Interfaces;
+
+use Illuminate\Http\Client\RequestException;
+
+interface RequestSender
+{
+    /**
+     * Make an API request to the ExchangeRatesAPI.
```

Comment:
```suggestion
     * Make an API request to the specified driver's API.
```

Reply:
Ahh yeah, good spot! If you want to make a PR for this bit so you can get some credit for helping me review the PR, I'll get that change merged in 😄